### PR TITLE
support to display running test metrics as charts or tables

### PR DESCRIPTION
**Pull request by lemoncrust**
added crude support to display running test metrics as terminal charts or a table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,15 @@ otgen transform
   [--file template.tmpl]              # Go template file. If not provided, built-in templates will be used based on provided parameters
 ````
 
+### `display`
+
+Displays metrics of a running test as charts or a table.
+
+```Shell
+otgen display
+  [--type charts|table]               # Display type: "charts" to show charts, "table" to show table
```
[the remark this answers is not in the document]
Comment (lemoncrust) [reply README.md]: done

File: BUILD.md [hunk 2]
```diff
@@ -70,4 +70,20 @@ cat test/transform/port_metrics.json | ./otgen transform -f templates/transformP
```
[the remark this answers is not in the document]
Comment (lemoncrust) [reply BUILD.md:71]: done

File: cmd/display.go [hunk 3]
```diff
@@ -77,5 +81,6 @@ func init() {
 	// Cobra supports local flags which will only run when this command
 	// is called directly, e.g.:
 	// transformCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
-	displayCmd.Flags().StringVarP(&displayType, "type", "t", "charts", "charts or table")
+	displayCmd.Flags().StringVarP(&displayType, "mode", "m", TYPE_CHART, fmt.Sprintf("display mode [%s|%s]", TYPE_CHART, TYPE_TABLE))
+	displayCmd.Flags().StringVarP(&chartType, "type", "t", "", "chart type [line] (default line)")
```
Comment (bortok) [inline cmd/display.go:85]: let's make `line` to be a default type explicitly here, via:
```
displayCmd.Flags().StringVarP(&chartType, "type", "t", "line", "chart type [line]")
```

Comment (bortok) [reply cmd/display.go:85]: that would also add `(default line)`  help string automatically
not sure if we want to bother checking for the value to be one of those we support

Comment (bortok) [reply cmd/display.go:85]: pls ignore as such approach would interfere with `--mode table`

**Pull request merged**